client/stylesheets/_responsive.scss

```diff
@@ -479,6 +493,21 @@
     }
 }
 
+@media (max-width: 375px) {
+    .profileCardInfo li:first-child {
+        margin-bottom: 6px !important;
+    }
+
+    .profileCardItem {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardItem` should be written in lowercase with hyphens